### ISSUE TITLE
OS Clubs Sitemap v1

### ISSUE THREAD

**Issue by m-branson** [mozilla/coss#213]
Based on the OS Clubs prototype and the current Campus Clubs website, I will build a proposed OS Clubs Sitemap. This sitemap will be an initial version, with intention being for this to evolve as content for, and functionality of, the website is iterated.

This Sitemap will provide a basis for a site content outline. This will be shared with the programmatic team for population and will help to determine project scope.

**Comment by m-branson** [comment]
OS Clubs site map is available for review via this [link](https://app.mural.ly/invitation/mural/osclubs0536/1501011620461?sender=mbranson7980&key=3232210884) (also attached inline for ease of viewing). 

This is from an 'ideal' experience perspective. Planning to make a Milestone 1 version based on user journey work once it is complete, that will be tracked via a separate ticket.

@a-kilroy for review.

![osc_sitemap_v1](https://user-images.githubusercontent.com/18686986/28699971-a66e3936-7301-11e7-9acd-f4e972f77ebb.jpg)

**Comment by a-kilroy** [comment]
As always, this is great.

A couple of small notes, I believe the 'ideal' includes some automation
around registration form to club page.

I also don't know if we have enough information to say what an ideal
experience is under resources besides FAQ. I'm fine leaving this as is but
if we can visually represent that somehow that would be awesome.

**Comment by m-branson** [comment]
Thanks @a-kilroy!

Agreed on the automation from club registration to club page, I've added a simple pathway there. See below.

As for the Resources section, I'd like to leave as is for now. From the service/ux standpoint 'ideal' is akin to the long term vision for the site. Now that we're narrowing in on the short term club captain user journey, I'll be making a truncated version of this sitemap that only includes short term milestones. For Resources I see that being:

1. Link to an external Onboarding resource (Teachable?)
2. Activities Page w/list of Activities and their external links
3. Curriculum -- this will not be present, any curriculum will be listed under Activities (unless the content proves a need otherwise)
4. FAQ as a page, rather than an external link.

One thing I find missing after our collective journey mapping exercises is a Project page. I'm currently mulling over whether this will live under Resources or as it's own 'Find A Project' page that lives as part of the main navigation. I'm leaning toward the later, what do you think?

Goal: By EOW have a truncated sitemap and base outline ready to share with the program side for population.

**Comment by m-branson** [comment]
Based on the Phase 1 Sitemap a [content outline](https://docs.google.com/a/mozilla.com/document/d/1KH32NLcMfsPE3lsE-OwHW2uXSCBjNVWHz79uk5BXGfI/edit?usp=sharing) has been generated. This will be populated by the program team prior to launch. Further IA needs will be tracked via new issues, closing.
